WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -19,7 +20,21 @@ final class ProductCategoryListViewController: UIViewController, GhostableViewCo
     let viewModel: ProductCategoryListViewModel
 
     private let configuration: Configuration
-    private var selectedListSubscription: AnyCancellable?
+    private var subscriptions: Set<AnyCancellable> = []
+
+    /// The controller of the view to show if the search results are empty.
+    ///
+    /// - SeeAlso: State.empty
```

Comment:
Sorry, I don't understand what this means. Can you please add more details to this line?

Reply:
This is actually redundant 😅 Removed in aa720905703c9cea54c7a78e7186e38f12e8a011.